integration-tests/load/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/rs/zerolog v1.30.0
 	github.com/slack-go/slack v0.12.2
 	github.com/smartcontractkit/chainlink-automation v1.0.2-0.20240118014648-1ab6a88c9429
-	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240219152510-85226a0fbdc1
+	github.com/smartcontractkit/chainlink-common v0.1.7-0.20240221153538-1ea85cf3dc6c
```

Comment:
Are we using some new import from chainlink-common?

Reply:
Anirudh is updating it here: https://github.com/smartcontractkit/chainlink/pull/12125/files